### DISCUSSION TITLE
Will t-pot work properly, if the vps doesn't accept nested virtualization via the SVM flag ?

### DISCUSSION THREAD

**Discussion by xgithx** [telekom-security/tpotce#1678] [Q&A]
First, I have installed Windows, on the vps, but in order for the t-pot to work on Windows, Docker Desktop had to be installed. In order for Docker Desktop to work properly, the vps, had to have nested virtualization via the SVM flag enabled. Unfortunately, the vps stuff said, the nested virtualization via the SVM flag is disabled on their vps, and they can not start it. So I can not use t-pot on Windows, via the vps in discussion, because Docker Desktop can not work without virtualization via the SVM flag.
After that, I have installed Ubuntu server on the vps .
Considering the the vps can not enable virtualization via the SVM flag, will I be able to use t-pot on Ubuntu, considering that t-pot can not work on Windows withoud Docker Desktop, and Docker Desktop can not work on Windows if on the vps virtualization via the SVM flag, it is not turned on.

**Comment by t3chn0m4g3** [comment]
Running T-Pot with Docker Desktop does not give you the full feature set anyway.
If you have the option to run a T-Pot supported Linux, Docker will run just fine.

**Reply by xgithx** [reply]
Thank you @t3chn0m4g3 
Can you tell me please, if t-pot will work as it should, if the vps, doesn't support virtualization via the SVM flag, where my installed Linux is.
Docker Desktop, on Windows, didn't work, because the vps didn't support virtualization via the SVM flag.
I appreciate.

**Reply by t3chn0m4g3** [reply]
Based on the discussion you already seem to have subscribed to the VPS, what is preventing you from just installing T-Pot and share your results?

**Comment by xgithx** [comment]
Because I have purchased a vps subscription just only for using t-pot...
I would really appreciate, if someone could tell me, asap, if T-pot will work on Ubuntu, hosted by a vps which doesn't support virtualization via the SVM flag. So, I could ask for a cancellation of the subscription, in order to benefit for a refund. @t3chn0m4g3 @adepasquale @adepasquale @glaslos @skoops please? Thank you.

**Comment by xgithx** [comment]
Anyone please ?

**Comment by glaslos** [comment]
Have you tried running docker on your VPS?
